src/extension/tsconfigHelper.ts

```diff
@@ -30,17 +31,20 @@ export class TsConfigHelper {
                 return fileSystem.readFile(tsConfigPath, "utf-8");
             })
             .then(function(jsonContents: string): Q.Promise<any> {
-                return JSON.parse(jsonContents);
+                return JSON.parse(stripJsonComments(jsonContents));
             });
     }
 
     /**
      * Writes out a JSON configuration object to the tsconfig.json file.
      */
     public static writeConfigJson(configJson: any): Q.Promise<void> {
+        return;
```

Comment:
Console.log won't work here, you'd need to use a vscode API to pop up a message at the top of the screen.

However I think a better approach is to avoid writing unless necessary. If I recall correctly, the reason that we originally wrote this tsconfig file was to get intellisense working across multiple JS files. If the user is making their own tsconfig file I think it is probably better to leave it untouched, so a more appropriate solution might be to only create the tsconfig if one doesn't exist, and otherwise don't bother doing anything to it.